modules/preview-react/radio/lib/StyledRadioButton.tsx

```diff
@@ -1,150 +1,218 @@
+      borderColor: base.licorice200,
       height: '18px',
       width: '18px',
       justifyContent: 'center',
       pointerEvents: 'none',
       position: 'absolute',
       transition: 'border 200ms ease, background 200ms',
-      opacity: disabled && variant === 'inverse' ? '.4' : '1',
-    },
-
-    '&:hover + .cnvs-radio-check': {
-      borderColor: disabled
-        ? inputColors.disabled.border
-        : variant === 'inverse'
-        ? colors.soap300
-        : colors.licorice500,
+      opacity: '1',
     },
 
-    '&:focus + .cnvs-radio-check': {
-      borderColor: disabled
-        ? inputColors.disabled.border
-        : variant === 'inverse'
-        ? colors.soap300
-        : colors.blueberry400,
-    },
-
-    '&:checked + .cnvs-radio-check': {
-      backgroundColor: variant === 'inverse' ? themePrimary.main : themePrimary.contrast, // inner circle color
-      border: `5px solid`, // this creates the inner circle
-      borderColor: variant === 'inverse' ? colors.frenchVanilla100 : themePrimary.main, // outer circle color
+    '&:hover + .cnvs-radio-check, &.hover + .cnvs-radio-check': {
+      borderColor: base.licorice500,
     },
 
-    '&:focus + .cnvs-radio-check, &:focus:hover + .cnvs-radio-check': {
-      outline: 'transparent',
-
+    '&:focus-visible + .cnvs-radio-check, &.focus + .cnvs-radio-check': {
+      borderColor: base.blueberry400,
       ...focusRing({
-        width: variant === 'inverse' ? 2 : 1,
+        width: 1,
         separation: 0,
         animate: false,
-        innerColor: variant === 'inverse' ? colors.blackPepper400 : colors.frenchVanilla100,
-        outerColor: variant === 'inverse' ? colors.frenchVanilla100 : themeFocusOutline,
+        innerColor: base.frenchVanilla100,
+        outerColor: brand.common.focusOutline,
       }),
     },
 
-    '&:focus:checked + .cnvs-radio-check, &:focus:hover:checked + .cnvs-radio-check': {
+    '&:focus-visible:hover + .cnvs-radio-check, &.focus:hover + .cnvs-radio-check': {
       outline: 'transparent',
-      ...focusRing({
-        width: 2,
-        separation: 2,
-        animate: false,
-        innerColor: variant === 'inverse' ? colors.blackPepper400 : colors.frenchVanilla100,
-        outerColor: variant === 'inverse' ? colors.frenchVanilla100 : themeFocusOutline,
-      }),
     },
 
-    ...mouseFocusBehavior({
-      '&:focus + .cnvs-radio-check, &:focus:hover + .cnvs-radio-check, &:focus:active + .cnvs-radio-check, &:focus:hover:checked + .cnvs-radio-check, &:focus:active:checked + .cnvs-radio-check':
-        {
-          ...focusRing({
-            width: 0,
-            outerColor: variant === 'inverse' ? colors.frenchVanilla100 : themeFocusOutline,
-          }),
+    '&:checked + .cnvs-radio-check, &.checked + .cnvs-radio-check': {
+      backgroundColor: brand.primary.accent, // inner circle color
+      border: `5px solid`, // this creates the inner circle
+      borderColor: brand.primary.base, // outer circle color
```

Comment:
Comment: These code comments don't make sense to me. It also seems like these repeated styles could be consolidated and shared.

```suggestion
      border: `${px2rem(5)} solid ${brand.primary.base}`,
```

Reply:
Agreed. Updated all borders